deepsubtree.go

```diff
@@ -0,0 +1,271 @@
+package iavl
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// Represents a IAVL Deep Subtree that can contain
```

Comment:
nit: I think your line wrap can be updated. Default is usually 80 characters. You code editor should have a setting for this or if you are a vim fan `gq`